packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -407,35 +409,44 @@ function createReactNoop(reconciler: Function, useMutation: boolean) {
 
   let yieldedValues = null;
 
+  let didYield;
   let unitsRemaining;
 
+  function shouldYield() {
+    if (
+      scheduledCallbackTimeout === -1 ||
+      elapsedTimeInMs > scheduledCallbackTimeout
```

Comment:
~~It's early and I'm probably being stupid, but shouldn't this be `elapsedTimeInMs < scheduledCallbackTimeout` ?~~

**Edit** Never mind. This is just saying that we should never yield if work has already timed out.